plumber/importer/__init__.py

```diff
@@ -45,6 +58,17 @@ def get_target_fps(self, context: Context) -> float:
         scene = context.scene
         return scene.render.fps / scene.render.fps_base
 
+    def is_batch_import(self) -> bool:
+        """Check if this is a batch import (multiple files selected)"""
```

Comment:
This is incorrect, files will always be populated so only 2 files and above should be considered a batch